### ISSUE TITLE
Quick Start v2: VoiceOver: references to "Tap" in toast messages should be read as "Double tap"

### ISSUE THREAD

**Issue by designsimply** [wordpress-mobile/WordPress-iOS#12129]
Steps to reproduce:

1. Go to Settings > General > Accessibility > VoiceOver and toggle the setting on.
1. Open a site in the app which has Quick Start v2 enabled or create a new site and select the option for quick start at the end of Site Creation ([screenshot](https://user-images.githubusercontent.com/1119271/61245040-d95f7700-a708-11e9-84ea-7ad966003389.png)).
1. Double tap any item.
1. Double tap the toast message that appears.

**Result:** references to "Tap" in toast messages should be read as “Double tap” by VoiceOver. ([6s](https://cloudup.com/czSRAhRo_Rm))

<img src="https://user-images.githubusercontent.com/1119271/61246134-60155380-a70b-11e9-89c5-cd07aa67c49a.png" width="270" alt="wpios-voiceover-quick-start-toast-message">
<sup>Tested with WP Internal 12.8.0.201907011 on iPhone 6S iOS 12.3.1.</sup>

<p style="clear:both;"><sub>(internal reference: p77Llu-bQ6-p2 h/t @etoledom)</sub></p>
